.github/workflows/ci.yml

```diff
@@ -53,8 +53,8 @@ jobs:
       - uses: wyvox/action-setup-pnpm@v3
         with:
           node-version: ${{ matrix.node-version }}
-      - name: ember test
-        run: pnpm test:ember
+      - name: tests
```

Comment:
we'll want to change this to not run lints: https://github.com/ember-template-imports/ember-template-imports/blob/master/package.json#L24